scripts/ci/pre_commit/pre_commit_check_integrations.sh

```diff
@@ -16,14 +16,11 @@
 # specific language governing permissions and limitations
 # under the License.
 set -euo pipefail
-
-PRE_COMMIT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-AIRFLOW_SOURCES=$(cd "${PRE_COMMIT_DIR}/../../../" && pwd);
-cd "${AIRFLOW_SOURCES}" || exit 1
-
 # shellcheck source=scripts/ci/libraries/_script_init.sh
 . "$( dirname "${BASH_SOURCE[0]}" )/../libraries/_script_init.sh"
 
+cd "${AIRFLOW_SOURCES}" || exit 1
```

Comment:
Is these variables defined in one of the file we source above?

Reply:
Yep